classes/DirtEvent.sc

```diff
@@ -84,8 +84,9 @@ DirtEvent {
 
 		if (~unit == \c) { speed = speed * ~unitDuration * ~cps };
 
-		endSpeed = speed * (1.0 + (accelerate.abs.linexp(0.01, 4, 0.001, 20, nil) * accelerate.sign));
-		if(endSpeed.sign != speed.sign) { endSpeed = 0.0 }; // never turn back
+		// endSpeed = speed * (1.0 + (accelerate.abs.linexp(0.01, 4, 0.001, 20, nil) * accelerate.sign));
+		// if(endSpeed.sign != speed.sign) { endSpeed = 0.0 }; // never turn back
+		endSpeed = speed * (1.0 + accelerate);
```

Comment:
good. What is a bit trickier is to make accelerate depend on dur. This is because the dur itself is defined in terms of speed and accelerate. It requires solving a quadratic equation, I think, so not too bad, but would need to be done with a bit of extra care to actually work for all the `unit`cases. I'm not sure it's worth the trouble ...